sample-functions/BaseFunctions/coffee/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+echo "Building functions/base:node-6.9.1-alpine"
+docker build -t functions/base:node-6.9.1-alpine .
```

Comment:
would you want that labeled with the coffeescript version? i.e. function/base:coffeescript-1.12-alpine?